### PR TITLE
Updated gitlab code host configuration

### PR DESCRIPTION
**Pull request by Rhia2**
## Test plan
No review required

### CODE REVIEW

File: doc/admin/external_service/gitlab.md [hunk 1]
```diff
@@ -4,10 +4,35 @@ Site admins can sync Git repositories hosted on [GitLab](https://gitlab.com) (Gi
 
 To connect GitLab to Sourcegraph:
 
-1. Go to **Site admin > Manage code hosts > Add repositories**
+1. Go to **Site admin > Manage code hosts > Add code host**
 2. Select **GitLab**.
-3. Configure the connection to GitLab using the action buttons above the text field, and additional fields can be added using <kbd>Cmd/Ctrl+Space</kbd> for auto-completion. See the [configuration documentation below](#configuration).
-4. Press **Add repositories**.
+3. Set **url** to the URL of your GitLab instance, such as https://gitlab.example.com or (for GitLab.com) https://gitlab.com.
```
Comment (indradhanush) [inline doc/admin/external_service/gitlab.md:9]: Nit: 

```suggestion
3. Set **url** to the URL of your GitLab instance, such as https://gitlab.example.com or https://gitlab.com (for GitLab.com).
```

File: doc/admin/external_service/gitlab.md [hunk 2]
```diff
@@ -4,10 +4,35 @@ Site admins can sync Git repositories hosted on [GitLab](https://gitlab.com) (Gi
...
+4. Create a GitLab access token using this [instructions](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token) with repo scope, and set it to be the value of the token.
```
Comment (indradhanush) [inline doc/admin/external_service/gitlab.md:10]: Nit:

```suggestion
4. Create a GitLab access token using these [instructions](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token) with repo scope, and set it to be the value of the token.
```

File: doc/admin/external_service/gitlab.md [hunk 3]
```diff
@@ -4,10 +4,35 @@ Site admins can sync Git repositories hosted on [GitLab](https://gitlab.com) (Gi
...
+5. Use the [Repository syncing documentation below](#repository-syncing) to select and add your preferred projects/repos to the configuration.
+6. You can use the action buttons above the text field to add the fields, and additional fields can be added using <kbd>Cmd/Ctrl+Space</kbd> for auto-completion. See the [configuration documentation below](#configuration) for additional fields.
+7. Click **Add repositories**.
+
+Example value:
```
Comment (indradhanush) [inline doc/admin/external_service/gitlab.md:15]:
```suggestion
Example config:
```

**Review by indradhanush — APPROVED** [review]
Thank you for improving the docs! ❤️

Left a few minor suggestions. Approved to unblock.

**Comment by indradhanush** [comment]
@sourcegraph/repo-management For a second pair of eyes.

File: doc/admin/external_service/gitlab.md [hunk 4]
```diff
@@ -4,10 +4,35 @@ Site admins can sync Git repositories hosted on [GitLab](https://gitlab.com) (Gi
 
 To connect GitLab to Sourcegraph:
 
-1. Go to **Site admin > Manage code hosts > Add repositories**
+1. Go to **Site admin > Manage code hosts > Add code host**
 2. Select **GitLab**.
```
Comment (sashaostrikov) [inline doc/admin/external_service/gitlab.md:8]:
```suggestion
2. Select **GitLab** (for GitLab.com) or **GitLab Self-Managed**.
```

**Review by sashaostrikov — APPROVED** [review]
LGTM! Added a suggestion.

**Pull request merged**